scripts/utils.py

```diff
@@ -0,0 +1,25 @@
+import sys
+from typing import Callable, Optional
+
+from python_on_whales import DockerClient
```

Comment:
`python_on_whales` documentation says

> Works on Linux, macOS and Windows, for Python 3.8 and above.

So maybe now we don't need to skip docker tests on windows and macos?